packages/adaptive_scaffold/README.md

```diff
@@ -1,18 +1,35 @@
 <?code-excerpt path-base="excerpts/packages/adaptive_scaffold_example"?>
 
-# Helper Widgets for Making Adaptive Layouts in Flutter (AdaptiveScaffold)
+# Helper Widgets for Making Adaptive Layouts in Flutter (`AdaptiveScaffold`)
```

Comment:
We can follow up in a separate PR for these README changes:

I would just change this to:
> # Adaptive Scaffold

since that's the package name.

Reply:
Done.